test/chunkify/chunkify_test.js

```diff
@@ -122,4 +120,20 @@ describe('Chunkify', function () {
         }
         done();
     });
+
+    /**
+     * Tests whether exception is thrown when invalid/negative CHAR_LIMIT and a text file converted to a String are passed into chunkify
+     */
+
+    it('should throw error when maxChars is less than 1', function(done) {
+        const testFilePath = "test/chunkify/testInputSmall.txt";
+        const VALID_SMALL_TEST_MESSAGE = readTextFile(testFilePath);
+        assert.throws(
+            () => {
+                chunkify(VALID_SMALL_TEST_MESSAGE, 0);
```

Comment:
could you group this together with the other invalid maxChars test

Reply:
Grouped in new commit